### ISSUE TITLE
Styling issue, a thin left border is showing in the field list configuration panel

### ISSUE THREAD

**Issue by azizaparveen** [Esri/solutions-components#537]
### Describe the bug

There is a thin gray border only on  the left side of the Fields list panel
![image](https://github.com/Esri/solutions-components/assets/138827632/42e6baea-e4f3-4bca-8688-9dd7c4dbd2d5)

### Expected Behavior

There should'nt be border on only one side

### Reproduction Steps & Sample

1) Open a manager app in devext in configuration mode
2) Click on Interactivity then Table options
3) Click on Field's button and notice the gray thin border on the left side

**Comment by jmhauck** [comment]
@azizaparveen seems reasonable to me as well but this is another one we should check with them on just in case its desired for some reason. This should be the same across other places where field config is available in other apps.

**Comment by sumitzarkar** [comment]
@azizaparveen 
This is an issue with the styles in the configuration of instant app. We don't see any option to fix this from component side.

@jmhauck  - please let us know your thoughts on this

**Comment by jmhauck** [comment]
Correct, this would be a change in configuration app not our components.
